ts/features/bonus/cgn/components/merchants/CgnMerchantsDiscountItem.tsx

```diff
@@ -42,19 +43,22 @@ const CgnMerchantDiscountItem: React.FunctionComponent<Props> = ({
             {index(0, [...discount.productCategories]).fold(
               undefined,
               categoryKey =>
-                getCategorySpecs(categoryKey).fold(undefined, c => (
-                  <View style={styles.row}>
-                    <IconFont
-                      name={c.icon}
-                      size={22}
-                      color={IOColors.bluegrey}
-                    />
-                    <View hspacer small />
-                    <H5 weight={"SemiBold"} color={"bluegrey"}>
-                      {I18n.t(c.nameKey).toLocaleUpperCase()}
-                    </H5>
-                  </View>
-                ))
+                getCategorySpecs(categoryKey as ProductCategoryEnum).fold(
```

Comment:
```suggestion
                getCategorySpecs(categoryKey).fold(
```
The same as before, I think this should not be required.

Reply:
ddde264 fixes